src/server.h

```diff
@@ -956,14 +958,11 @@ typedef struct replBufBlock {
     char buf[];
 } replBufBlock;
 
-/* Opaque type for the Slot to Key API. */
-typedef struct clusterSlotToKeyMapping clusterSlotToKeyMapping;
-
 /* Redis database representation. There are multiple databases identified
  * by integers from 0 (the default database) up to the max configured
  * database. The database number is the 'id' field in the structure. */
 typedef struct redisDb {
-    dict *dict;                 /* The keyspace for this DB */
+    dict **dict;                /* The keyspace for this DB */
     dict *expires;              /* Timeout of keys with a timeout set */
```

Comment:
I believe we originally wanted to avoid this since it would make the `volatile *` eviction policies much more difficult. I think that was before we came up with the idea with the fenwick trees, so this might be a lot simpler to implement now.